src/monitor/btcaddrs.go

```diff
@@ -0,0 +1,206 @@
+package monitor
+
+import (
+	"encoding/json"
+	_ "encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/boltdb/bolt"
+	"github.com/sirupsen/logrus"
+	"github.com/skycoin/teller/src/addrs"
+	"github.com/skycoin/teller/src/util/httputil"
+)
+
+type BtcAddressJSON struct {
+	Addresses []string `json:"btc_addresses"`
+}
+
+type CountBtc struct {
+	All  int
+	Used int
+	Free int
+}
+
+func openDB() (*bolt.DB, error) {
+	db, err := bolt.Open("test.db", 0600, nil)
+	if err != nil {
+		return nil, err
+	}
+	return db, err
+}
+
+// method: POST
+// url: /addbtc  {"btc_addresses": ["1st_add", "2nd_adr", ...]}
+var AddBtcAddresses = func(w http.ResponseWriter, r *http.Request) {
```

Comment:
No need for `var`, `func FuncName` is fine

Reply:
you are right, fix.